jobs/test_e2e_errand/templates/bin/run.erb

```diff
@@ -15,40 +16,63 @@ export PATH="/var/vcap/packages/java8/bin:$PATH"
 
 [ ! -e /home/vcap/.sincedb ] || rm /home/vcap/.sincedb
 
-
-echo '' ; echo '==> Purging existing test data...'
-
-curl -s -X DELETE '<%= p('test_e2e_errand.elasticsearch_hostport') %>/_all/sample-logs--repo-commits' ; echo ''
```

Comment:
My understanding is that Delete by Query has been depreciated as of 1.5.3 - https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-delete-by-query.html#docs-delete-by-query
and I can't figure out a straight forward way to replace it.

I also couldn't get this specific delete to work; I think because when I ship the logs they don't get parsed with a `@type:sample-logs--repo-commits`

```
"_source":{"@version":"1","@timestamp":"2015-07-14T17:31:10.427Z","host":"10.12.8.148","@type":"syslog","@message":"<14>1 2015-07-14T17:31:10.427+0000 sample-logs--repo-commits LOGSTASH - - - 9d666a0da317777d35e763800ddfd8507cfb580c b056061ab74cc30ee2e6ea3e530f7262b2245228 2015-05-28 18:36:28 -0600 Danny Berger // Start experimenting with concourse","syslog_pri":["14","14"],"syslog5424_ver":1,"syslog_program":["LOGSTASH","LOGSTASH"],"syslog_message":"9d666a0da317777d35e763800ddfd8507cfb580c b056061ab74cc30ee2e6ea3e530f7262b2245228 2015-05-28 18:36:28 -0600 Danny Berger // Start experimenting with concourse","received_at":["2015-07-14T17:31:10.472Z","2015-07-14T17:31:10.427Z"],"received_from":["10.12.8.148","10.12.8.148"],"tags":["syslog_standard","syslog_standard"],"syslog_severity_code":6,"syslog_facility_code":1,"syslog_facility":"user-level","syslog_severity":"informational","@source.host":"sample-logs--repo-commits","syslog_procid":["-","9d666a0da317777d35e763800ddfd8507cfb580c","-"],"syslog_msgid":["-","b056061ab74cc30ee2e6ea3e530f7262b2245228","-"]}'
```

Which log_parsing rules are you using?